pyproject.toml

```diff
@@ -110,6 +110,8 @@ ignore = [
     "ANN", # Ignore type annotations in tests
     "S101",  # Yeah of course we assert in tests
     "B009",  # Allow calling `getattr` in tests since it can be used to make the test clearer.
+    "S103", # Allow `os.chmod` setting a permissive mask `0o555` on file or directory
+    "S108", # Allow Probable insecure usage of temporary file or directory
```

Comment:
It's better to not include these project-wide, and instead add a noqa in the test where the 555 chmod is used, and refactor the plugin tests to not set `part_info._part_install_dir` and test against `new_dir / "parts/foo/install"` instead.

Reply:
It doesn't show on the default diff but these ignores only affect the `"tests/**.py"` files:
![image](https://user-images.githubusercontent.com/1951299/211311755-d3becc74-e82c-4a13-a983-7a3c890c4295.png)

Reply:
do you think we should update the test code regardless?

Reply:
If it's only for tests it should be fine.